Documentation & Methods Download Page/docdownload.Rmd

```diff
@@ -0,0 +1,56 @@
+---
+title: "Measure & Methodology Documentation Download"
+output: 
+  html_document:
+    theme: cosmo
+    toc: true
+    toc_depth: 4
+    toc_float: true
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+Each measure used in the MWI Tool was assessed based on its strengths, limitations, and theoretical justification for inclusion into the tool. Detailed documentation for each can be accessed below. All methods used to calculate unqiue statistical procedures can also be accessed below.
```

Comment:
Add something about clicking to download.

Reply:
👌